src/System.Drawing.Design.Primitives/src/SR.cs

```diff
@@ -0,0 +1,25 @@
+using System.Resources;
+
+namespace System
```

Comment:
Please don't add SR.cs files in CoreFx.  Port the code to understand the common SR and its usage.  (I know I did this in WinForms/WPF codebase but that was to create a minimal diff with the desktop code since I knew you'd have to maintain code flow for a period of time).